engine/fixtureParser.js

```diff
@@ -19,6 +19,11 @@ export default class FixtureParser {
 
   readFile(src) {
     const meta = matter.read(src);
+
+    if (Object.keys(meta.data).length === 0) {
+        throw new Error(`Error while parsing '${src}'.`);
```

Comment:
Can we provide a better explanation? Something like:
'Error while parsing ... . Metadata is empty but it is mandatory.'